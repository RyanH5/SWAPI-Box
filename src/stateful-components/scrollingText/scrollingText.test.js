import React from 'react';
import ReactDOM from 'react-dom';
import ScrollingText from './index';
import {shallow} from 'enzyme';

describe('ScrollingText', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<ScrollingText />, 
      { disableLifecycleMethods: true }); 
  });

  // it('should match a snapshot', () => {
  //   expect(wrapper).toMatchSnapshot();
  // });
});