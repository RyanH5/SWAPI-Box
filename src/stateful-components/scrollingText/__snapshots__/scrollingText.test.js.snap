// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ScrollingText should match a snapshot 1`] = `undefined`;
